### PR TITLE
Styling for the Phoenix home page

### PR DESCRIPTION
**Pull request by paulcsmith**
Based on #169 Content still needs work, but I mostly **focused on the styling**. We can **tweak content once the design is live**.

#### This PR
* Adds base rename.io html, css and JavaScript
* Adds Bourbon for mixins. Right now I'm not using much, but I will likely use it more in the future.
* Adds basic README to explain how to generate the CSS

**A lot of this PR is not my code**. The following are the custom css files that I need reviewed!!
* custom.scss
* variables.scss
* hero.scss

screen shot 2015-04-24 at 3 30 28 pm

#### Things I'd still like to do (later?)

* Move the custom styles in "Phoenix_files" to "stylesheets"
* See if I can clean up the HTML! It's pretty nuts right now, but that is how Firefox saved it.
* See if I can use more specific selectors to get rid of the `!important`s in the CSS.

#### Next steps

* Merge this PR in
* Update custom styles on readme.io (from the generated `custom.css`)
* Get SVG version of the logo, make it white and upload it to Readme.io
* Update the HTML to use the buttons html found in `Phoenix.html`
* Fix anything that breaks (we can't do this before hand since there is no "preview" for custom styles")
* Add other button for screencast (once it's ready)

### CODE REVIEW

File: phoenix-landing-page/Phoenix_files/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
+@import url(http://fonts.googleapis.com/css?family=Lato:300,400,700);
```
Comment (paulcsmith) [inline phoenix-landing-page/Phoenix_files/_hero.scss]: This file contains the bulk of the CSS I did

**Comment by lancehalvorsen** [comment]
(From the Krakow airport, 6:50AM) 

Thanks for all this!

I had created the "styling" directory at the root level of this repo for just this. How would you feel about moving the whole "phoenix-landing-page" directory inside of it?

**Comment by paulcsmith** [comment]
Oh yes! I'll definitely do that. Thanks for the heads up!

**Comment by lancehalvorsen** [comment]
:+1:

File: phoenix-landing-page/Phoenix_files/_hero.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+body, h1, h2, h3, h4, h5, h6 {
+  font-family: 'Lato', sans-serif !important;
+}
+
+.header {
+  @include radial_gradient(circle at 50% 50%, #F67938 29%, #EF4734 120%);
```
Comment (garbles) [inline phoenix-landing-page/Phoenix_files/_hero.scss]: how do you feel about moving `#F67938` and `#EF4734` to variables? (same for other colors in this file)

**Comment by garbles** [comment]
This looks awesome!

**Comment by garbles** [comment]
The Phoenix image in the top left looks a little blurry in your screen shot. Should we see about opting for an SVG?

**Comment by paulcsmith** [comment]
@garbles Thanks! Yes, I'm getting an SVG version from Chris so it'll be a lot smoother :)

File: phoenix-landing-page/Phoenix_files/_hero.scss (hunk 2 above)
Comment (paulcsmith) [inline phoenix-landing-page/Phoenix_files/_hero.scss]: Fantastic idea. I missed those!

**Comment by garbles** [comment]
@paulcsmith :clap::grin:

File: phoenix-landing-page/Phoenix_files/_hero.scss (hunk 2 above)
Comment (garbles) [inline phoenix-landing-page/Phoenix_files/_hero.scss]: :family:

**Comment by paulcsmith** [comment]
This is now live! This has the latest CSS and can be merged in if it looks good to you @lancehalvorsen

**Comment by lancehalvorsen** [comment]
Most excellent! Thanks so much for this @paulcsmith!

**Pull request merged**